bricks.js

```diff
@@ -115,6 +115,46 @@ class Brick { // type 0 = invis, 1 = brick, 2 = question, 3 = block
             ctx.strokeRect(this.bottomBB.x - this.game.camera.x, this.bottomBB.y, this.bottomBB.width, this.bottomBB.height);
         }
     };
+
+    explode(){
+        this.game.addEntity(new Shard(this.game, this.x, this.y, -150));
+        this.game.addEntity(new Shard(this.game, this.x, this.y  - PARAMS.BLOCKWIDTH * 1.5, -150));
+        this.game.addEntity(new Shard(this.game, this.x + PARAMS.BLOCKWIDTH, this.y - PARAMS.BLOCKWIDTH * 1.5, 150));
+        this.game.addEntity(new Shard(this.game, this.x + PARAMS.BLOCKWIDTH, this.y, 150));
+        this.removeFromWorld = true;
+    }
+};
+
+class Shard{
+    constructor(game, x, y, xVelocity, yVelocity = -100){
+        Object.assign(this, {game, x, y})
+
+        this.animation = new Animator(ASSET_MANAGER.getAsset("./sprites/bricks.png"), 16, 0, 8, 8, 1, 0.33, 0, false, true);
+        this.velocity = {x:xVelocity, y:yVelocity};
+        this.life = 90; //90 clock ticks -> 1.5 seconds at 60 fps
+
+    }
+
+    update(){
+        const FALL_ACC = 562.5;
+        this.life -= 1;
```

Comment:
We generally want to use this.game.clockTick when doing time based events so that no matter what the FPS is the entities timing is consistent.